### PR TITLE
:bug: [warning] Fix warnings on index

### PR DESCRIPTION
**Pull request by JoelRomero97**
This PR fixes warnings while rendering because of possibly undefined variables as well as using `ReactDOM.render` instead of `ReactDOM.createRoot` (react 18)

Screenshot 2022-12-19 at 11 22 00 AM

Screenshot 2022-12-19 at 11 27 10 AM

Screenshot 2022-12-19 at 11 30 17 AM

### CODE REVIEW

File: demo/web/index.html [hunk 1]
```diff
@@ -3,11 +3,16 @@
   <head>
     <meta charset="utf-8">
     <title></title>
-    <script src="https://unpkg.com/react/umd/react.development.js"></script>
-    <script src="https://unpkg.com/react-dom/umd/react-dom.development.js"></script>
-    <script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
-    <script src="https://cdn.jsdelivr.net/npm/axios@0.18.0/dist/axios.min.js"></script>
-    <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css"></link>
+    <script 
+src="https://unpkg.com/react/umd/react.development.js"></script>
```
Comment (whummer) [inline demo/web/index.html:7]: Can we please undo the formatting changes in the PR, or at least fix the formatting, such that multi-line tags / script lines contain proper indentation? 🙏

**Review by whummer — COMMENTED** [review]
Thanks for this PR @JoelRomero97 ! Can we please fix the indentation before we start a more detailed review? Thanks!

File: demo/web/index.html (hunk 1 above)
Comment (JoelRomero97) [reply demo/web/index.html]: No idea what happened while pushing, done

**Review by whummer — APPROVED** [review]
LGTM, thanks for adding these fixes @JoelRomero97 ! 🚀

**Pull request merged**